R/variable_label.R

```diff
@@ -122,7 +122,7 @@ assign_label.data.frame <- function(x, value, ...){
   }
 
   for(i in seq_along(colnames(x))) {
-    if(colnames(x)[i] %in% names(value)) {
+    if(any(colnames(x)[i] == names(value))) {
```

Comment:
Just out of curiosity, what's the advantage of this approach?

Reply:
Hadley recommends using this construct because it is by magnitudes faster